next/package.json

```diff
@@ -18,7 +18,7 @@
     "@types/react": "18.2.21",
     "@types/react-dom": "18.2.7",
     "autoprefixer": "10.4.15",
-    "clsx": "^2.1.0",
+    "clsx": "^2.1.1",
```

Comment:
Are we still using clsx in other places, or was `EventCard` the only one? If it's totally gone now we might be able to remove this from package.json